src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -779,7 +779,8 @@ private IEnumerator RecordFrame(bool endRecording = false, bool endRecordingFrom
                             screenSize = new Vector2Int() { x = screenWidth, y = screenHeight },
                             performance = performanceMetrics,
                             pixelHash = pixelHash,
-                            currentRenderPipeline = GraphicsSettings.currentRenderPipeline.GetType().FullName,
+                            // In Unity, if the GraphicsSettings.currentRenderPipeline property is null, the default render pipeline is the Built-in Render Pipeline.
+                            currentRenderPipeline = GraphicsSettings.currentRenderPipeline == null ? null : GraphicsSettings.currentRenderPipeline.GetType().FullName,
```

Comment:
Handles null `currentRenderPipeline`, which means.. 'default render pipeline' in Unity

Reply:
I debated doing that, but what 'null' means to Unity could very well change.  I decided on writing the actual value.